src/components/ha-selector/ha-selector-action.ts

```diff
@@ -19,13 +27,39 @@ export class HaActionSelector extends LitElement {
 
   @property({ type: Boolean, reflect: true }) public disabled = false;
 
+  @state()
+  @consume({ context: fullEntitiesContext, subscribe: true })
+  _entityReg: EntityRegistryEntry[] | undefined;
+
+  @state() private _entitiesContext;
+
+  protected hassSubscribeRequiredHostProps = ["_entitiesContext"];
+
   private _actions = memoizeOne((action: Action | undefined) => {
     if (!action) {
       return [];
     }
     return migrateAutomationAction(action);
   });
 
+  protected firstUpdated() {
+    if (!this._entityReg) {
+      this.hass.loadBackendTranslation("device_automation");
```

Comment:
This should probably be moved to the device automation action? Or is it also used in other parts?

Reply:
Is there a function that tests if this has been loaded? Or will it no-op if it has been loaded?

If I put this in the device automation action I'm not sure how I will know if it has been previously loaded.

Reply:
It should no-op it if it has been loaded already. Should be handled by the translation loading logic.